app/services/model_exporters/team_exporter.rb

```diff
@@ -3,11 +3,8 @@
 module ModelExporters
   class TeamExporter < ModelExporter
     def initialize(team_id)
-      @team = Team.includes(:user_teams).find_by_id(team_id)
-      raise StandardError, 'Can not load team' unless @team
```

Comment:
same here

Reply:
.find() throws an exception if object not found